containers/addresses/AddressModal.tsx

```diff
@@ -50,6 +56,7 @@ const AddressModal = ({ onClose, member, organizationKey, ...rest }) => {
         }
 
         const { Address } = await api(
+            // @ts-ignore - TODO better typings
```

Comment:
I'd rather add `/** @type {(any => any)} */` above `createAddress` in  addresses.js

When we eventually rewrite that file to typescript, that comment will be removed and any errors regarding parameters will be visible in the project, with ts-ignore, they won't and we'd have to remember to do a lookup for them.

Reply:
Ok I'll think about this next time. I already added ts-ignore in a bunch of places. I have it in my list to do this lookup later